### PR TITLE
Join study group prompt login

### PR DESCRIPTION
**Pull request by railsstudent**
When unauthenticated user attempts to join study group in all_study_groups page, error message is displayed in upper right corner of the page.

Make improvement to prompt sign in modal instead of error message.  User can click sign in button to login with Slack subsequently.

join_study_group_prompt_login

### CODE REVIEW

File: client/templates/study_groups/all_study_groups.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ Template.allStudyGroups.onCreated(function() {
   const title = "CodeBuddies | Study Groups";
   const metaInfo = {
     name: "description",
-    content: "CodeBuddies is a community of independent code learners who enjoy sharing knowledge and helping each other learn faster. We come from all over the world; there are members living in the United States, Japan, Sweden, the United Kingdom, Russia, Australia, Canada, India, and more. Everyone is welcome, independent of previous knowledge."
+    content:
```
[the remark this answers is not in the document]
Comment (lpatmo) [reply client/templates/study_groups/all_study_groups.js]: I believe the newline was created automatically by prettier (https://github.com/prettier/prettier), a package we installed in the last release. It's made some code formatting improvements to our files for new PRs we submit invisibly, and doesn't show up until the PR is submitted.

[the remark this answers is not in the document]
Comment (sergeant-q) [reply client/templates/study_groups/all_study_groups.js]: hrrmm. ok I see. They did say it's a very opinionated code formatter after all!

[the remark this answers is not in the document]
Comment (lpatmo) [reply client/templates/study_groups/all_study_groups.js]: :P

[the remark this answers is not in the document]
Comment (distalx) [reply client/templates/study_groups/all_study_groups.js]: We should add `i18` files to `prettierignore`.

[the remark this answers is not in the document]
Comment (lpatmo) [reply client/templates/study_groups/all_study_groups.js]: Agree that we could add the i18n files to .prettierignore.

**Pull request merged**